apps/omg_child_chain_rpc/test/omg_child_chain_rpc/web/plugs/health_test.exs

```diff
@@ -26,15 +26,19 @@ defmodule OMG.ChildChainRPC.Plugs.HealthTest do
       :ok = :alarm_handler.clear_alarm(@alarm_2)
       :ok = :alarm_handler.set_alarm(@alarm_1)
 
-      pull_client_alarm(
-        300,
-        %{
-          "data" => %{
-            "code" => "operation:service_unavailable"
-          }
-        },
-        fn -> TestHelper.rpc_call(:post, "/block.get", %{}) end
-      )
+      :ok =
+        pull_client_alarm(
+          300,
+          %{
+            "data" => %{
+              "code" => "operation:service_unavailable",
+              "description" => "The server is not ready to handle the request.",
```

Comment:
We're too specific here, we'll need to change this test everytime we fix spelling or...

Reply:
But this is a *unit test*. That's exactly what the test is testing for!

Reply:
So if I accidentally change the description to "yolo what is this" and leave the code the same ("operation:service_unavailable"). Which test will cover for this bug?

Reply:
In that `yolo` case specific controller test will fail. C'mon Ino here you're testing `Health` plug not that controller response has specific format.

Reply:
No it will not. The `plug` kicks in before the controller and the plug is producing the response.
https://github.com/omisego/elixir-omg/blob/master/apps/omg_child_chain_rpc/lib/omg_child_chain_rpc/web/plugs/health.ex#L97

Reply:
Reponded below :point_down: